src/Button/Flip/Flip.component.tsx

```diff
@@ -1,25 +1,36 @@
 // VENDOR
 import * as React from 'react';
-import styled from '@xstyled/styled-components';
-import { th } from '@xstyled/system';
+import styled, { css } from '@xstyled/styled-components';
+
 // ANCHOR
 import { Theme, TRANSITION_SPEED, REVEAL_BACKGROUND_COLOR } from '../utils';
 
 export interface StyledFlipProps extends React.HTMLAttributes<HTMLDivElement> {
+    circular?: boolean;
     flipColor: string;
 }
 
 export const StyledFlip = styled('div')<StyledFlipProps>`
-    border-top-right-radius: ${th.radius('base')};
-    border-bottom-left-radius: ${th.radius('base')};
-
     position: absolute;
     top: -1px;
     right: -1px;
-    width: 1.25rem;
-    height: 1.25rem;
     content: '';
 
+    ${({ circular }) =>
+        circular
+            ? css`
+                  border-top-right-radius: 1000px;
```

Comment:
if `base` works below, then `circular` should work here.

Reply:
Ah yes, I didn't realize `circular` was in the theme. It looks like I have more work to do in this PR. Circular flip only works in the `md` size. The problem is that I want to make the flip be 50% of the button height and make it a square. I haven't thought of a good way to do that yet.

Reply:
make it a square on a circular button? do you have a mock?

`height` is something you can pass into the flip component from the button component, then just `calc(${height} / 2)` should work I think